src/components/casehistory/ChargeHistoryStats.js

```diff
@@ -18,90 +24,126 @@ import {
-      numFelonyCharges,
-      numFelonyConvictions,
-      numViolentCharges,
-      numViolentConvictions
-    } = getSummaryStats(chargeHistory);
+  const statusText = pendingCharges.size
+    ? `${pendingCharges.size} Pending Charge${pendingCharges.size > 1 ? 's' : ''}`
+    : 'No Pending Charges';
+  const mode = pendingCharges.size ? 'danger' : 'neutral';
 
-    const SUMMARY_STATS_ARR = [
-      {
-        label: '# of misdemeanor charges',
-        value: numMisdemeanorCharges
-      },
-      {
-        label: '# of felony charges',
-        value: numFelonyCharges
-      },
-      {
-        label: '# of violent charges',
-        value: numViolentCharges
-      },
-      {
-        label: '# of misdemeanor convictions',
-        value: numMisdemeanorConvictions
-      },
-      {
-        label: '# of felony convictions',
-        value: numFelonyConvictions
-      },
-      {
-        label: '# of violent convictions',
-        value: numViolentConvictions
-      }
-    ];
+  const {
+    numMisdemeanorCharges,
+    numMisdemeanorConvictions,
+    numFelonyCharges,
+    numFelonyConvictions,
+    numViolentCharges,
+    numViolentConvictions
+  } = getSummaryStats(allCharges, chargeIdsToSentenceDates);
 
-    const SummaryStats = SUMMARY_STATS_ARR.map((stat) => (
-      <StatsItem key={stat.label}>
-        <StatLabel>{stat.label}</StatLabel>
-        <StatValue>{stat.value}</StatValue>
-      </StatsItem>
-    ));
+  const SUMMARY_STATS_ARR = [
+    {
+      label: '# of misdemeanor charges',
+      value: numMisdemeanorCharges
+    },
+    {
+      label: '# of felony charges',
+      value: numFelonyCharges
+    },
+    {
+      label: '# of violent charges',
+      value: numViolentCharges
+    },
+    {
+      label: '# of misdemeanor convictions',
+      value: numMisdemeanorConvictions
+    },
+    {
+      label: '# of felony convictions',
+      value: numFelonyConvictions
+    },
+    {
+      label: '# of violent convictions',
+      value: numViolentConvictions
+    }
+  ];
 
-    return (
-      <StatsWrapper padding={padding}>
-        <StatsSectionHeader>
-          <Title withSubtitle>
-            <span>Summary Statistics</span>
-            All current and past cases
-          </Title>
-          {pendingCharges.size ? this.renderPendingChargeStatus() : null}
-        </StatsSectionHeader>
-        <StatsContainer>
-          <StatsSubWrapper>
-            <StatsGroup>
-              {SummaryStats}
-            </StatsGroup>
-          </StatsSubWrapper>
-        </StatsContainer>
-      </StatsWrapper>
-    );
-  }
-}
+  const SummaryStats = SUMMARY_STATS_ARR.map((stat) => (
+    <StatsItem key={stat.label}>
+      <StatLabel>{stat.label}</StatLabel>
+      <StatValue>{stat.value}</StatValue>
+    </StatsItem>
+  ));
+
+  return (
+    <StatsWrapper padding={padding}>
+      <StatsSectionHeader>
+        <Title withSubtitle>
+          <span>Summary Statistics</span>
+          All current and past cases
```

Comment:
Why not also wrap with `<span>`?

Reply:
I think Title applies certain styles to span when passes `withSubtitle` prop, so that is intentional.